File: Projet_final/projetFinalSpringBoot/src/main/java/com/exemple/model/Adresse.java
package com.exemple.model;

import javax.persistence.Column;
import javax.persistence.Embeddable;

@Embeddable
public class Adresse {
	@Column(name = "adresse_numero")
	private Integer numero;
	@Column(name = "adresse_rue")
	private String rue;
	@Column(name = "adresse_code_postal")
	private String codePostal;
	@Column(name = "adresse_ville")
	private String ville;
	@Column(name = "adresse_pays")
	private String pays;

	public Integer getNumero() {
		return numero;
	}

	public void setNumero(Integer numero) {
		this.numero = numero;
	}

	public String getRue() {
		return rue;
	}

	public void setRue(String rue) {
		this.rue = rue;
	}

	public String getCodePostal() {
		return codePostal;
	}

	public void setCodePostal(String codePostal) {
		this.codePostal = codePostal;
	}

	public String getVille() {
		return ville;
	}

	public void setVille(String ville) {
		this.ville = ville;
	}

	public String getPays() {
		return pays;
	}

	public void setPays(String pays) {
		this.pays = pays;
	}
}

File: Projet_final/angularBT/src/app/model/adresse.ts
export class Adresse {
  
    constructor(private _numero:string='',
  private _rue:string='',
  private _codePostal:string='',
  private _ville:string='',
  private _pays:string=''){
    
  }
  
  public get rue(): string {
    return this._rue;
  }
  
  public set rue(_rue: string) {
    this._rue =_rue;
  }
  
  public get numero(): string {
    return this._numero;
  }
  
  public set numero(_numero: string) {
    this._numero =_numero;
  }
  
  public get codePostal(): string {
    return this._codePostal;
  }
  
  public set codePostal(_codePostal: string) {
    this._codePostal =_codePostal;
  }
  
  public get ville(): string {
    return this._ville;
  }
  
  public set pays(_pays: string) {
    this._pays =_pays;
  }
  
  public get pays(): string {
    return this._pays;
  }
  
  public set ville(_ville: string) {
    this._ville =_ville;
  }

}